code/obj/machinery/firealarm.dm

```diff
@@ -92,8 +93,19 @@
 /obj/machinery/firealarm/process()
 	if(status & (NOPOWER|BROKEN))
 		return
-
 	use_power(10, ENVIRON)
+	secondary_tick++
+	if(secondary_tick > 3)
```

Comment:
I recall Naksu working on variable machine ticking. Might be us both looking into.

Reply:
Yep, you just set a var.